README.md

```diff
@@ -197,6 +197,46 @@ The above command `pip3 install --user -r requirements.txt` installs the latest
 export PYTHONPATH=SOME_PATH/PyGHee:$PYTHONPATH
 ```
 
+### <a name="step4.2"></a>Step 4.2: Installing tools to access S3 bucket
+
+The script `scripts/eessi-upload-to-staging` uploads a tarball and an associated metadata file to an S3 bucket. It needs two tools for this, `aws` to actually upload the files and `jq` to create the metadata file. This section describes how these tools are installed and configured on the `bot machine`.
+
+Create a new directory, say `BOT_ROOT/tools` and change into the directory.
+
+For installing the AWS Command Line Interface, including the tool `aws`,
+please follow the instructions at the
+[AWS Command Line Interface guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```

Comment:
It's probably worth highlighting that the `eessi-upload-to-staging` script is also compatible with `awscli` v1.x, which is way easier to install (via `pip install`, see https://pypi.org/project/awscli) than v2.x...

Reply:
Address via #111